tests/test_modeling_common.py

```diff
@@ -2652,7 +2653,7 @@ def get_current_gpu_memory_use():
         config = self.model_tester.get_large_model_config()
 
         for model_class in self.all_parallelizable_model_classes:
-            torch.cuda.empty_cache()
+            backend_empty_cache(torch_device)
```

Comment:
it's ok to me to keep this change. So even if we still use `require_torch_xxx_gpu`, we try to use `backend_xxx`, is this what you are trying to do?

Reply:
@ydshieh yes, i am proposing to use all device-agnostic backend APIs, so device specific test cases can be identified only by the decorator(@require_torch_gpu, @require_torch_accelerator, @require_torch_multi_gpu, @require_torch_multi_accelerator etc.), currently in some cases we use decorator to identify, in some cases we use both, i think decorator-only is enough and make it more convenient to extend the cases to other capable devices in the future. Pls let me know your comments to this proposal, thx.